R/groupingsets.R

```diff
@@ -60,7 +60,7 @@ groupingsets.data.table = function(x, j, by, sets, .SDcols, id = FALSE, jj, ...)
     stop("All columns used in 'sets' argument must be in 'by' too. Columns used in 'sets' but not present in 'by': ", brackify(setdiff(sets.all.by, by)))
   if (id && "grouping" %chin% names(x))
     stop("When using `id=TRUE` the 'x' data.table must not have a column named 'grouping'.")
-  if (any(sapply(sets, anyDuplicated)))
+  if (any(vapply_1i(sets, anyDuplicated)))
```

Comment:
shouldn't this one be `vapply_1b`?

Reply:
I forgot `anyDuplicated` returns integer index of first dup, `0L` otherwise.  So `_1i` is correct then.